src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -207,6 +207,7 @@ internal static int GetWarningLevel(ErrorCode code)
             switch (code)
             {
                 case ErrorCode.WRN_OptionalParamValueMismatch:
+                case ErrorCode.WRN_ParamsArrayInLambdaOnly:
```

Comment:
Why are we using warning level 8?
Warning levels greater than 4 are for diagnostics that we want to introduce but break compatibility. But the two warnings here are only affecting new lambdas, so don't need to use a "warning wave" warning. They can use warn level 1.

Reply:
Makes sense, changed: 28a9338c9b5341d7ea3f5535c7c8262fc1b1f3c6

Before this PR, it made sense to have a new warning level since `WRN_OptionalParamValueMismatch` was raised also for method groups (existing code).